### PR TITLE
The Grit House Classic

### PR DESCRIPTION
**Pull request by TrystanOakley**
@sstangl

### CODE REVIEW

**Comment by TrystanOakley** [comment]
The Travis said that Tn was not recognized as a state? @sstangl

**Comment by sstangl** [comment]
It's expecting the state to be all-caps, "TN".

**Comment by TrystanOakley** [comment]
Oh my bad I'll fix it

File: meet-data/spf-archive/0436/lifters.csv [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+117.5,Open,Justin Jones,31,106.59,TN,229.06,188.24,238.14,655.44,M,Wraps,SBD,2
+117.5,Open,Cameron Gordon,30,117.48,TN,229.06,156.49,278.96,664.51,M,Wraps,SBD,1
+125,Open,Michael Bales,29,122.47,TN,288.03,226.8,299.37,814.2,M,Wraps,SBD,1
+140,Open,Kyle Coleman,,131.54,TN,301.64,158.76,299.37,759.77,M,Wraps,SBD,1
+100,Masters (40-44),John Greeves,42,108.86,GA,192.78,149.69,235.87,578.33,M,Wraps,SBD,1
+117.5,Masters (40-44),Chris Moorse,41,117.48,TN,272.16,172.37,235.87,680.39,M,Wraps,SBD,1
+82.5,Masters (45-49),David Russell,45,81.19,TN,217.72,151.95,226.8,596.47,M,Wraps,SBD,1
+90,Masters (45-49),Larry Berry,49,89.36,TN,224.53,165.56,226.8,616.89,M,Wraps,SBD,1
+82.5,Masters (55-59),Arthur McCann,58,88,GA,122.47,99.79,204.12,426.38,M,Wraps,SBD,1
+82.5,Masters (45-49),David Russell,45,81.65,TN,217.72,,,217.72,M,Wraps,SBD,2
```
Comment (sstangl) [inline meet-data/spf-archive/0436/lifters.csv]: David Russell was Squat-only here, so the Event should be "S". I'm surprised this passed tests! I gotta make the tests better...

File: meet-data/spf-archive/0436/lifters.csv [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+100,Open,Shane Everage,30,98.88,KY,,206.38,294.84,501.22,M,Raw,SBD,1
```
Comment (sstangl) [inline meet-data/spf-archive/0436/lifters.csv]: Shane Everage was push-pull, so the Event should be "BD".

File: meet-data/spf-archive/0436/lifters.csv [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+125,Teenage (18-19),Dalton Swanger,18,122.47,TN,,210.92,,210.92,M,Raw,SBD,1
```
Comment (sstangl) [inline meet-data/spf-archive/0436/lifters.csv]: Dalton through Thomas Odell are Bench-only, so "B".

File: meet-data/spf-archive/0436/lifters.csv [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+100,Sub-masters,Todd Morgan,45,99.79,TN,,217.72,,217.72,M,Raw,SBD,1
+100,Open,Tony Evans,,99.79,GA,,226.8,,226.8,M,Raw,SBD,2
+110,Open,James Faust,25,108.41,TN,,226.8,,226.8,M,Raw,SBD,1
+100,Open,Robert Patterson,42,97.98,GA,,181.44,,181.44,M,Raw,SBD,3
+110,Masters (55-59),Ken Hancock,55,106.59,TN,,165.56,,165.56,M,Raw,SBD,1
+82.5,Masters (70-74),John Vedepo,73,,NC,,97.52,,97.52,M,Raw,SBD,1
+110,Masters (70-74),Thomas Odell,70,,TN,,145.15,,145.15,M,Raw,SBD,1
+100,Masters (65-69),Jimmy Bishop,67,95.71,MS,,,161.03,161.03,M,Raw,SBD,1
+82.5,Masters (50-54),Scott Alford,52,80.74,TN,278.96,183.7,263.08,725.75,M,Single-ply,SBD,1
+60,Teenage (18-19),Devin Everage,19,58.97,KY,,72.57,142.88,215.46,M,Single-ply,SBD,1
```
Comment (sstangl) [inline meet-data/spf-archive/0436/lifters.csv]: Same Event stuff with the equipped folk.

**Comment by TrystanOakley** [comment]
I'm an idiot my bad. I'm home visiting parents. Accidentally made it all
SBD it seems

**Comment by sstangl** [comment]
No problem. I updated the test suite to catch Event errors, and it found 2,048 of them. So, uh, I guess that happens a lot. Probably not going to check in that patch :|

**Comment by TrystanOakley** [comment]
Check in that patch? What do you mean? If you need help fixing anything I'm
game.

**Comment by TrystanOakley** [comment]
Im in this for the long run lol

**Comment by TrystanOakley** [comment]
@sstangl it all should be good to go

**Comment by sstangl** [comment]
Thanks! :-)

**Pull request merged**